boards/arm/twr_ke18f/doc/index.rst

```diff
@@ -66,6 +66,12 @@ features:
 +-----------+------------+-------------------------------------+
 | FLASH     | on-chip    | soc flash                           |
 +-----------+------------+-------------------------------------+
+| I2C(M)    | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| SENSOR    | off-chip   | fxos8700 polling;                   |
+|           |            | (fxos8700 trigger;)                 |
```

Comment:
why the parentheses?  Maybe instead:
```
fxos8700 polling; trigger supported
with H/W mods explained below
```

Reply:
@dbkinder Thank you for your feedback. I have updated the documentation with your suggested change.